buildSrc/build.gradle

```diff
@@ -116,6 +116,7 @@ dependencies {
   api 'org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.10'
   api 'de.thetaphi:forbiddenapis:3.3'
   api 'com.avast.gradle:gradle-docker-compose-plugin:0.15.2'
+  api 'org.yaml:snakeyaml:1.32'
```

Comment:
Please `${versions.snakeyaml}`, from https://github.com/opensearch-project/OpenSearch/blob/main/buildSrc/version.properties

Reply:
Oh yes! Updated to use from the version.properties file